content/docs/specs/native_histograms.md

```diff
@@ -0,0 +1,2416 @@
+---
+title: Native Histograms [EXPERIMENTAL]
+sort_rank: 6
+---
+
+# Native Histograms
+
+Native histograms were introduced as an experimental feature in November 2022.
```

Comment:
It feels more like 0.1 (or 0.0.1 ;). Once we have addressed all (or at least most of) the TODOs here, and declared NH a stable feature, we might think about versioning. Right now, things are just too much in flux. (There are the git hashes, in any case.)

But I'm not even sure if there should be one version for all of NH. There is the exposition format (with its own versioning), there is RW (with its own versioning), there are chunks (with their own versioning), etc.

Reply:
Perhaps make it 1.0-rc.0 then, like we iterate over PRW 2.0? https://prometheus.io/docs/specs/remote_write_spec_2_0/ 🤔

Reply:
I would argue that version numbers aren't the right thing for this pseudo spec. It's way too large of a surface to version. As said above, we should version formats and protocols, not this entire beast.

(I'm also somewhat hesitant to add this to the "specification" section. It could as well be in the "concepts" section. Not saying I want to change it now, just to stress the point that this is only partially a spec.)